src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AssemblyBasedCallerUtils.java

```diff
@@ -178,4 +181,11 @@ public static AssemblyRegion assemblyRegionWithWellMappedReads(final AssemblyReg
                 .forEach(result::add);
         return result;
     }
+
+    public static VariantContext makeMergedVariantContext(final List<VariantContext> vcs) {
+        final List<String> sources = vcs.stream().map(VariantContext::getSource).collect(Collectors.toList());
```

Comment:
maybe it's a commonly used term that I'm just not familiar with. TODO: discuss with @davidbenjamin